concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -5,35 +5,28 @@
 import random
 import time
 from base64                 import b64encode
+from collections import namedtuple
 from freezegun              import freeze_time
 
 from golem_messages         import message
 
 from utils.helpers          import get_current_utc_timestamp
 from utils.testing_helpers  import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
+from api_testing_common import api_request, parse_command_line
+from api_testing_common    import timestamp_to_isoformat
 
 import requests
 
+HOUR = 3600
```

Comment:
I don't think this is necessary. First, you are using it only once. Second, after all it is hour, its value shouldn't change (if it can, it should be renamed).
But if you want the value `3600` you can use `int(timedelta(hours=1).total_seconds())`.

Reply:
I disagree. Well named constant is batter than some magic naumber, even if it is used only once. As to calculation, I am pretty sure I can handle it without timedelta ;)

Reply:
I just don't see a difference between this and, for example `ONE = 1`. Do you expect this value to ever change ? If yes, then it makes sense, but then it should be called differently, more meaningful in context of the test.